src/test/javascript/spec/component/iris/settings/iris-common-sub-settings-update.component.spec.ts

```
@@ -139,41 +144,41 @@ describe('IrisCommonSubSettingsUpdateComponent Component', () => {
-        expect(comp.allowedIrisModels).toEqual(comp.getAvailableModels());
+        comp.inheritAllowedVariants = true;
+        comp.onInheritAllowedVariantsChange();
+        expect(comp.subSettings!.allowedVariants).toBeUndefined();
+        expect(comp.allowedVariants).toEqual(comp.getAllowedVariants());
 
-        comp.inheritAllowedModels = false;
-        comp.onInheritAllowedModelsChange();
-        expect(comp.subSettings!.allowedModels).toEqual(comp.allowedIrisModels.map((model) => model.id));
+        comp.inheritAllowedVariants = false;
+        comp.onInheritAllowedVariantsChange();
+        expect(comp.subSettings!.allowedVariants).toEqual(comp.allowedVariants.map((model) => model.id));
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)